crates/bevy_ecs/src/world/mod.rs

```diff
@@ -2753,6 +2753,95 @@ impl World {
         Some(result)
     }
 
+    /// Temporarily removes the requested resource from this [`World`] if it exists, runs custom user code,
+    /// then re-adds the resource before returning. Returns the result of the closure.
+    ///
+    /// This enables safe simultaneous mutable access to both a resource and the rest of the [`World`].
+    /// For more complex access patterns, consider using [`SystemState`](crate::system::SystemState).
+    ///  /// # Example
```

Comment:
```suggestion
    ///
    /// # Example
```
Two points: you had an extra `///`, and you were missing a newline before the heading. I suspect these are related!